### PR TITLE
all changes to seed and query postgres

### PR DESCRIPTION
**Pull request by chanceknguyen**

### CODE REVIEW

File: database/generate.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+      if ( i === 0 ) {
+        stringifier.write({
+          id: counter++,
+          userGameId: counter - 1,
+          recommended: (Math.random() > 0.5) ? 1 : 0,
+          body: body,
+          helpful_count: Math.floor(Math.random() * 50),
+          funny_count: Math.floor(Math.random() * 10),
+          comments_count: Math.floor(Math.random() * 10),
+          awards: "{\"Treasure\":1,\"Mind Blown\":0,\"Golden Unicorn\":0,\"Deep Thoughts\":0,\"Heartwarming\":2,\"Hilarious\":3,\"Hot Take\":3,\"Poetry\":2,\"Extra Helpful\":1}",
```
Comment (michaelrtobias) [inline database/generate.js:140]: Im guessing all of the \ are for turning it into a nested JSON afterwards?

File: database/generate.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  await gameGen()
+  await userGen();
+  // await userGameGen();
+  await reviewsGen();
+}
+
+generateAll();
+
+
+  // const userGameGen = () => {
```
Comment (michaelrtobias) [inline database/generate.js:184]: Im guilty of this but should be deleting commented out stuff before committing, unless it has a purpose for later

File: database/generate.js (hunk 1 above)
Comment (chanceknguyen) [reply database/generate.js:140]: Yes, that's exactly it. It properly escapes it so it remains a nested JSON object.

**Review by michaelrtobias — APPROVED** [review]
Glad your queries are making it in time! other than cleaning up the code a bit, everything looks swell!

**Pull request merged**